flow/src/test/java/com/sequenceiq/flow/service/flowlog/FlowChainLogServiceTest.java

```diff
@@ -171,6 +172,7 @@ private FlowChainLog flowChainLog(String flowChanId, boolean hasEventInQueue, Lo
             flowEventChain.add(new TestEvent());
         }
         flowChainLog.setChain(JsonWriter.objectToJson(flowEventChain));
+        flowChainLog.setChainJackson(TypedJsonUtil.writeValueAsStringSilent(flowEventChain));
```

Comment:
If I get it right, this will only verify the `null != latestFlowChain.getChainJackson()` branch (relying on `TypedJsonUtil.readValueWithJsonIoFallback()`), but skip the `else` branch (using `JsonReader.jsonToJava()`).

Reply:
covered both cases